rootfs/usr/local/bin/s3

```diff
@@ -20,7 +20,7 @@ if [[ "$ACTION" = "fstab" ]]; then
 	# Create mount point directory
 	mkdir -p "${MOUNT_POINT}"
 elif [[ "$ACTION" = "mount" ]]; then
-	mount -a
+	if ! mountpoint -q "${MOUNT_POINT}"; then mount -a; fi
```

Comment:
Move this fix to s3fs wrapper script.

Reply:
(that way `mount -a` won't fail)

Reply:
Did you mean `rootfs/usr/local/bin/init-terraform` script where `s3` command is used?
Or something like this (in file `rootfs/usr/bin/s3fs`):
```bash
if ! mountpoint -q "${MOUNT_POINT}"; then
    exec goofys --file-mode=0600 --dir-mode=0700 --region=${BUCKET_REGION} --sse --acl=private ${DEVICE##s3://} ${MOUNT_POINT}
fi
```

Reply:
it seems to me that the second one is more reasonable..

Reply:
Objective is that `mount -a` should not fail. Thus, fix belongs in `rootfs/usr/bin/s3fs` which is called by `mount`